pkgs/os-specific/linux/mwprocapture/default.nix

```diff
@@ -14,15 +11,15 @@ let
 
 in
 stdenv.mkDerivation rec {
-  name = "mwprocapture-1.2.${version}-${kernel.version}";
-  version = "4177";
+  name = "mwprocapture-1.3.0.${version}-${kernel.version}";
+  version = "4236";
 
   src = fetchurl {
     url = "http://www.magewell.com/files/drivers/ProCaptureForLinux_${version}.tar.gz";
```

Comment:
```suggestion
    url = "https://www.magewell.com/files/drivers/ProCaptureForLinux_${version}.tar.gz";
```

Reply:
I think that https here doesn't give us any advantages? [Debian mirrors](https://www.debian.org/CD/http-ftp/#mirrors) rarely provide https, and the signing in use there is less robust than a pre-known hash like we have. https may instead be an added risk for failed builds if the source has certificate or other configuration problems.

While I prefer http, if that is best practice or convention on NixOS I don't mind changing it. I looked at the [manual](https://nixos.org/manual/nixpkgs/stable/#chap-pkgs-fetchers) and the example there gives http, for what it's worth.

Reply:
Just use https if there is https. The contents of Debian mirrors are fully signed or anyone could do whatever with them.

> https may instead be an added risk for failed builds if the source has certificate or other configuration problems.

There is always a risk that the download somehow doesn't work. If that happens we can think about it at the time.

Reply:
Very well, thanks.